Wiki/resources/resources-conda-setup.md

```diff
@@ -1 +1,135 @@
+```
+/g/data/hh5/public/apps/miniconda3/envs/analysis3-22.10/bin
+```
+becomes
+```
+/g/data/hh5/public/apps/conda-scripts/analysis3-22.10.d/bin
+```
+There are also environment variables set that are interpreted by the launcher script required to access commands within the squashfs conda environment. 
+
+Every entry in the in-container `envs/analysis3-22.10/bin` directory is symlinked to the launcher script placed in the `analysis3-22.10.d/bin` directory. These links are programmatically generated during installation, and invoke singularity with the correct squashfs mounted, and then inspect the link name to determine which command execute the command from within the container. The launcher script also has provisions for command overrides and configuration to alter the behaviour of commands in the container if necessary. The launcher script can also be invoked directly in order to run arbitrary commands inside the container. For example, running `launcher.sh bash` will launch an interactive shell inside the container with all correct bind, overlay and squashfs mounts in place.
+
+The actual container used as the base of the environment contains only enough components to create a functional environment. The container does not contain its own operating system, instead, it is comprised of a series of empty directories and symlinks. The necessary components of Gadi's operating systems are bind-mounted in at launch time through the launcher script. Though this does make the environment entirely unportable, which goes against the philosophy of containerisation, the container itself only needs to be constructed once for any given system, and reconstruction is trivial. The advantage of this approach is that the conda environment is separate to the container, and therefore multiple conda environments can be present 'in' the same container. This also means that the container can never be out of sync as Gadi's OS receives updates. This allows us to make modifications to the conda environment after installation that enhance the functionality of the environment. 
+
+As a part of the installation process, the `openssh` packages are removed from the conda installation, which forces use of the system `ssh` and, more importantly, its configuration. The `openmpi` package is also replaced by the `openmpi` distribution installed in `/apps` on Gadi. This ensures that the conda environment has a fully compatible MPI distribution with the host, which solves the problem of conda installing incorrectly configured MPI distributions, and addresses the drawbacks outlined in the [Singularity documentation](https://docs.sylabs.io/guides/3.3/user-guide/mpi.html).
```

Comment:
No problem with this, but it might be nice to highlight it, partially just to break the text, partially as it's such a great advantage of this system.
A way to do it is to use an admonition block, similar to the note and warning above but where you can set your own title
```{admonition} title
content
```